base.Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM debian:stretch
```

Comment:
Is there any reason you can't just use the prebuilt openjdk base image? E.g. `openjdk:8-stretch`:
https://hub.docker.com/layers/openjdk/library/openjdk/8-stretch/images/sha256-6896e96ce729fd54da25b8d831400773efeef39780c65bffb2d64c1412189455?context=explore

Reply:
I'll try that image and see if it works